pyvista/plotting/plotter.py

```diff
@@ -4592,6 +4604,13 @@ def update_scalars(self, scalars, mesh=None, render=True):
         render : bool, default: True
             Force a render when True.
         """
+        # Deprecated on 0.43.0, estimated removal on v0.50.0
```

Comment:
Three versions of a minor release to be deprecated is sufficient in my mind.
```suggestion
        # Deprecated on 0.43.0, estimated removal on v0.46.0
```

Reply:
I'm good with that. I figured this API has been around for a long time and wanted to give folks plenty of time. 3 releases ~= 9 months, right?

Reply:
Your right. Thanks.